Yoast/wordpress-seo
# Use specific tinyMCE check

## Summary

This PR can be summarized in the following changelog entry:

* Disables the Markers when editting a content item in Gutenberg.

## Relevant technical choices:

* Check for expected tinyMCE content ID

## Test instructions

This PR can be tested by following these steps:

* Open a post in Gutenberg
* Make sure there are markers in the classic editor
* See the markers not being displayed in the Gutenberg display

## Quality assurance


Fixes #
